### ISSUE TITLE
hydra.errors.OverrideParseException: mismatched input '=' expecting <EOF>

### ISSUE THREAD

**Issue by Cello2195** [MolecularAI/Chemformer#38]
When I was running fine_tune.sh, I came across the error "hydra.errors.OverrideParseException: mismatched input '=' expecting <EOF>". My shell file is as below:

#!/bin/bash

export HYDRA_FULL_ERROR=1

python -m molbart.fine_tune \
  datamodule=[molbart.data.seq2seq_data.Uspto50DataModule] \
  data_path=data/mol_opt.pickle \
  model_path=models/mask/step=1000000.ckpt \
  vocabulary_path=bart_vocab_downstream.json \
  task=backward_prediction \
  n_epochs=100 \
  learning_rate=0.001 \
  schedule=cycle \
  batch_size=64 \
  acc_batches=4 \
  augmentation_probability=0.5

**Comment by anniewesterlund** [comment]
The reason is likely because there is an equal sign in you model_path. Instead of writing
`model_path=models/mask/step=1000000.ckpt`, you can try
`'model_path=models/mask/step=1000000.ckpt'`
